recipes/extra-cmake-modules/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class ExtracmakemodulesConan(ConanFile):
+    name = "extra-cmake-modules"
+    license = "GPLv2"
```

Comment:
Please check this again, it does not look accurate

https://github.com/KDE/extra-cmake-modules/search?q=SPDX-License-Identifier

Looks like it may use several licenses, but for most modules it uses `BSD-3-Clause`. Please list all of them

Also please use the SPDX license names: https://spdx.org/licenses/